libs/libideviceactivation/Makefile

```diff
@@ -0,0 +1,99 @@
+#
+# Copyright (C) 2012-2024 OpenWrt.org
```

Comment:
please either assign the copyright to yourself or simply remove it
the same applies below

Reply:
I removed it. I giveaway any copyright in hope to improve OpenWRT and help others. 😀
I also adjusted license file and version to make sure they match each package. I just noticed they vary.